parser/src/layout.rs

```diff
@@ -173,6 +173,7 @@ where
         loop {
             // Retrieve the current indentation level if one exists
             let offside = match (&token.value, self.indent_levels.last().cloned()) {
+                (&Token::ShebangLine(_), _) => return Ok(token),
```

Comment:
Sorry about that. I never thought about the layered approach to tokenizer + layout may be hard to discover. Had I figured you would do this change in the parser rather than as a preprocess I would have warned you (probably).

Stuffing it front and centre in the token tests may help but it feels wrong to inject something that should be irrelevant for the tests themselves there... Wonder if there is a cleaner way to explain it